tests/Module/Fixture/Events.php

```diff
@@ -294,26 +286,88 @@ public function createEventsWithRelatedUserAndUser(int $owner, array $events)
             ];
 
             $eventId = $this->laravel->haveRecord(self::TABLE_EVENTS, $recordEvents);
-            $this->laravel->seeRecord(self::TABLE_EVENTS, $recordEvents);
 
             $recordRelatedUsers = [
-                'owner_id' => $owner,
+                'owner_id' => $ownerId,
                 'event_id' => $eventId,
                 'user_id' => $userId ?? '',
                 'related_user_id' => $temporaryUserId ?? '',
                 'created_at' => $createdAt,
             ];
-            
+
             $this->laravel->haveRecord(self::TABLE_RELATED_USERS, $recordRelatedUsers);
-            $this->laravel->seeRecord(self::TABLE_RELATED_USERS, $recordRelatedUsers);
 
             $users[] = $userId ?? $temporaryUserId;
             unset($userId);
             unset($temporaryUserId);
         }
     }
 
-    public function createEventsWithRelatedUserAndUserForExperimentStats(int $owner, array $events)
+    public function createRevenueEventsWithRelatedUserAndUser(
```

Comment:
what ? Why it's "revenue" events? Have you seen any mention of "revenue" below ?